### PR TITLE
feat: Export getOutgoingMessageState util function

### PR DESCRIPTION
**Pull request by HoonBaek**
## For Internal Contributors

[UIKIT-2159](https://sendbird.atlassian.net/browse/UIKIT-2159)

## Description Of Changes

* Export the `getOutgoingMessageState` utils function
* Remove `getOutgoingMessageStates`
* Add tests for it

## Types Of Changes

What types of changes does your code introduce to this project?
Put an `x` in the boxes that apply_

- [ ] Bugfix
- [x] New feature
- [ ] Documentation (correction or otherwise)
- [ ] Cosmetics (whitespace, appearance (ex) Prettier)
- [ ] Build configuration
- [ ] Improvement (refactor code)

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
     "@typescript-eslint/ban-ts-comment": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-unused-expressions": "error",
-    "react/display-name": "off"
+    "react/display-name": "off",
+    "no-unused-vars": "off",
```
Comment (sravan-s) [inline .eslintrc.json:22]: not a good idea to remove these rules 😉

File: src/utils/exports/__tests__/exportingUtils.spec.ts [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import { GroupChannel } from '@sendbird/chat/groupChannel';
+import { UserMessage } from '@sendbird/chat/message';
+import { getOutgoingMessageState, OutgoingMessageStates } from '../getOutgoingMessageState';
+
+describe('Global-export-utils-getOutgoingMessageState', () => {
+  test('OutgoingMessageStates should contain each types', () => {
+    expect(OutgoingMessageStates.NONE).toBe('NONE');
+    expect(OutgoingMessageStates.PENDING).toBe('PENDING');
+    expect(OutgoingMessageStates.SENT).toBe('SENT');
+    expect(OutgoingMessageStates.FAILED).toBe('FAILED');
```
Comment (sravan-s) [inline src/utils/exports/__tests__/exportingUtils.spec.ts:10]: 👍

File: .eslintrc.json (hunk 1 above)
Comment (HoonBaek) [reply .eslintrc.json:22]: ok ok

File: src/utils/exports/getOutgoingMessageState.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  SENT = 'SENT',
+  FAILED = 'FAILED',
+  DELIVERED = 'DELIVERED',
+  READ = 'READ',
+}
+
+export const getOutgoingMessageState = (channel: GroupChannel | OpenChannel, message: UserMessage | FileMessage): string => {
+  if (message.sendingStatus === 'pending') return OutgoingMessageStates.PENDING;
+  if (message.sendingStatus === 'failed') return OutgoingMessageStates.FAILED;
+  if (channel?.isGroupChannel?.()) {
```
Comment (sravan-s) [inline src/utils/exports/getOutgoingMessageState.ts:17]: use curly braces for `if else`  even one line please, is more readable ~

File: exports.js [hunk 4]
```diff
@@ -18,6 +18,9 @@ export default {
   'handlers/UserEventHandler': 'src/lib/handlers/UserEventHandler.ts',
   'handlers/SessionHandler': 'src/lib/handlers/SessionHandler.ts',
 
+  // utils
+  'utils/message/getOutgoingMessageState': 'src/utils/exports/getOutgoingMessageState.ts',
```
Comment (sravan-s) [inline exports.js:22]: will be nice if we follow same folder structure in utils, good fro future ->
```
src/utils/message/getOutgoingMessageState.ts
```

**Review by sravan-s — APPROVED** [review]

**Pull request merged**